serverless/aws/lambda-tests/src/main/scala/sttp/tapir/serverless/aws/lambda/tests/LambdaHandler.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.tapir.serverless.aws.lambda.tests
+
+import cats.effect.IO
+import com.amazonaws.services.lambda.runtime.{Context, RequestStreamHandler}
+import io.circe.Printer
+import io.circe.generic.auto._
+import io.circe.parser.decode
+import io.circe.syntax._
+import sttp.model.StatusCode
+import sttp.tapir.serverless.aws.lambda._
+
+import java.io.{BufferedWriter, InputStream, OutputStream, OutputStreamWriter}
+import java.nio.charset.StandardCharsets
+
+object LambdaHandler extends RequestStreamHandler {
```

Comment:
we need our own runtime as part of the main sources :) so that it isn't repeated here

Reply:
Ok, I'll replace that, there's is a configuration option in `sam local` to set environment variables